vega_sim/scenario/fuzzed_markets/scenario.py

```diff
@@ -518,28 +519,28 @@ def configure_agents(
                     ]
                 )
             ]
-            market_agents["fuzzy_reward_funders"] = [
-                FuzzyRewardFunder(
-                    wallet_name="REWARD_FUNDERS",
-                    key_name=f"MARKET_{str(i_market).zfill(3)}",
-                    asset_name=asset_name,
-                    step_bias=1,
-                    validity_bias=0.8,
-                    attempts_per_step=10,
-                    tag=f"MARKET_{str(i_market).zfill(3)}",
-                )
-            ]
-            market_agents["fuzzy_governance_transfer_agents"] = [
-                FuzzyGovernanceTransferAgent(
-                    wallet_name="REWARD_FUNDERS",
-                    key_name=f"MARKET_{str(i_market).zfill(3)}",
-                    asset_name=asset_name,
-                    step_bias=1,
-                    validity_bias=0.99,
-                    attempts_per_step=10,
-                    tag=f"MARKET_{str(i_market).zfill(3)}",
-                )
-            ]
```

Comment:
Deliberately commented out?

Reply:
Noooo, thank you good catch. Commented out for speed of testing.